Sources/NIO/EventLoop.swift

```diff
@@ -596,6 +602,10 @@ extension EventLoop {
     public func preconditionInEventLoop(file: StaticString = #file, line: UInt = #line) {
         precondition(self.inEventLoop, file: file, line: line)
     }
+
+    public var isOpen: Bool {
+        true
```

Comment:
This won't compile in our set of Swift versions: we need the explicit `return` statement.